Tests/SwiftParserTest/DeclarationTests.swift

```diff
@@ -1551,6 +1551,16 @@ final class DeclarationTests: XCTestCase {
       """
     )
   }
+
+  func testSuppressedImplicitConformance() {
+    assertParse(
+      """
+      struct Hello: ~Copyable {}
+
+      enum Whatever: Int, ~ Hashable, Equatable {}
+      """
+    )
```

Comment:
Could you create separate `assertParse` checks for these two cases and make sure they are parsed correctly (i.e. actually have the `~` as a separate token that’s not part of the identifier?